spec/data/dns/resolv.conf

```diff
@@ -0,0 +1,5 @@
+# Generated by NetworkManager
```

Comment:
You can probably put all of this content into a heredoc in the spec file since it is so small.